taxonomies/categories.txt

```diff
@@ -80829,6 +80858,21 @@ ciqual_food_code:en:28858
 ciqual_food_name:en:Pancetta, dried
 ciqual_food_name:fr:Pancetta ou Poitrine roulée sèche
 
+<en:Pancetta
+en:Arrotolata pancetta
+
+<en:Pancetta
+en:Coppata pancetta
+
+<en:Pancetta
+en:Steccata pancetta
+
+<en:Pancetta
+en:Stufata pancetta
+
+<en:Pancetta
+en:Tesa pancetta
```

Comment:
```suggestion
en:Tesa pancetta
es:Tesa pancetta
```

Reply:
Hello @benbenben2 here the 'en:' tag is already present. Do you want to add the 'es:' tag? In Spanish, it should say Pancetta tesa though. I'll ignore this suggestion for now (but I'll add the 'it:' tag since it's an italian product).